lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -76,188 +76,22 @@ def define
           if response == "none"
             outcome :outcome_not_affected
           else
-           question benefits.fetch(benefit_types.shift, :housing_benefit_amount?)
+            question benefits.fetch(benefit_types.shift, :housing_benefit_amount?)
```

Comment:
I prefer to fix these sort of minor problems in the commit that introduced them, i.e. amend the original commit so that the problem was never there in the first place.